pkg/printers/internalversion/describe.go

```diff
@@ -1235,6 +1241,12 @@ func describePersistentVolumeClaim(pvc *api.PersistentVolumeClaim, events *api.E
 		}
 		w.Write(LEVEL_0, "Capacity:\t%s\n", capacity)
 		w.Write(LEVEL_0, "Access Modes:\t%s\n", accessModes)
+		if pvc.Spec.VolumeMode != nil {
+			w.Write(LEVEL_0, "VolumeMode:\t%v\n", *pvc.Spec.VolumeMode)
+		} else {
+			var volumeMode api.PersistentVolumeMode
+			w.Write(LEVEL_0, "VolumeMode:\t%v\n", volumeMode)
```

Comment:
omit all the else blocks for things like this... just print the field if returned and non-nil

Reply:
thanks. I understand.
done.